lab2.js

```diff
@@ -44,25 +44,41 @@ function assert(expression, failureMessage) {
     fourth hour     |    4/hour        |        4         |
 
  TODO: First, make a constructor function, called Blob, that makes blobs.
-
  TODO: Next, create an instance of Blob named blob.
-
  TODO: Then, use a loop to calculate how long it took the blob to finish
- with Dowington. */
+ with Dowington.
+*/
+ var dowington = 1000;
+
+ function Blob () {
+  this.personsConsumed = 0;
+  }
 
+  var blob = new Blob();
+  for (var i = 0; i < dowington; i++) {
+    blob.personsConsumed += i;
+    if (blob.peopleConsumed > dowington) {
```

Comment:
Your asserts aren't passing because of a typo here - you have peopleConsumed instead of personsConsumed. Try fixing the typo and see what happens.